src/App.tsx

```diff
@@ -15,49 +23,82 @@ export const goodsFromServer = [
   'Garlic',
 ];
 
+interface IGoodsQuery {
+  sortBy: string;
+  isReversed: boolean;
+}
+
+const goods: IGoods[] = goodsFromServer.map(name => (
+  {
+    key: getRandomKey(),
+    name,
+  }
+));
+
+function getPreparedGoods(
+  goodsToPrepared: IGoods[],
+  {
+    sortBy,
+    isReversed,
+  }: IGoodsQuery,
```

Comment:
Why are you passing the second and third arguments as one object?
What is the point in this?

Reply:
@FrontKid I agree that this approach will help to avoid problems in the future, but then you have to pass all the arguments as an object, and not just a part, because now the arguments partially continue to depend on the order.